### DISCUSSION TITLE
Can I use malvis in production in minikube?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18395] [Q&A and General discussion]
Can I use malvis in production in minikube on one PC.
Is it wise?  Or should I deploy a full blown k8s cluster ?

What is the prefered and most tested way to install milvus?

**Comment by martinenkoEduard** [comment]
when using with driver=none - does it have limits on memory and cpu ?
How do I change it?

**Comment by yhmo** [comment]
The v2.0.2 has some critical bugs, I think it is not qualify for production. The v2.1 is more stable. 
Just pick a suitable way to deploy milvus according to your requirement. Just keep a mind that the milvus is a distributed system, it is scalable.
If you deployed milvus as cluster mode, it is recommended that each node's cpu core >= 2
